experiment/config-forker/main.go

```diff
@@ -252,6 +252,38 @@ func fixEnvVars(vars []v1.EnvVar, version string) []v1.EnvVar {
 	return newVars
 }
 
+func fixTestgridAnnotations(annotations map[string]string, version string, isPresubmit bool) map[string]string {
+	r := strings.NewReplacer(
+		"master-blocking", version+"-blocking",
+		"master-informing", version+"-informing",
+	)
+	a := map[string]string{}
+	didDashboards := false
+annotations:
+	for k, v := range annotations {
+		switch k {
+		case "testgrid-dashboards":
+			v = r.Replace(v)
+			if !isPresubmit {
+				v += ", " + "sig-release-" + version + "-all"
+			}
+			didDashboards = true
+			break
+		case "testgrid-tab-name":
+			v = strings.ReplaceAll(v, "master", version)
+			break
+		case "description":
+			continue annotations
+		}
+		a[k] = v
+	}
+	if !didDashboards && !isPresubmit {
+		a["testgrid-dashboards"] = "sig-release-" + version + "-all"
```

Comment:
Not sure I want to add a whole bunch more jobs to -all dashboards, but let's see how many this ends up being.  Conceivably if we get here, we've encountered a job that nobody is actually watching

Reply:
For now I just mimicked how this worked before, punting questions over whether this dashboard should exist at all to the future.